website/static/js/logTextParser.js

```diff
@@ -25,6 +25,23 @@ function ravenMessage (message, logObject) {
     }
 }
 
+/**
+ * Utility function to convert absolute URLs to relative urls
+ * See:
+ *      http://stackoverflow.com/questions/736513/how-do-i-parse-a-url-into-hostname-and-path-in-javascript
+ * This method have no effect on external urls.
+ * @param url {string} url to be converted
+ * @returns {string} converted relative url
+ */
+function toRelativeUrl(url) {
```

Comment:
This seems like a fairly universal function. We should probably add it to osfHelpers.js. Sorry for not noticing before.